assets/js/membraneWebRTC.ts

```diff
@@ -115,6 +120,11 @@ export interface Callbacks {
   onNewPeerTracks?: (peer: Peer, trackIdToMetadata: Map<string, any>) => void;
 }
 
+interface TrackWrapper {
+  trackId: string;
+  metadata: any;
+}
```

Comment:
It doesn't make sense for me because we don't have id at the moment of calling addTrack, which will not allow us to create trackId. So we will need another map e.g. localTrackIdToTrackContext what in my opinion is not a good idea.